.travis.yml

```diff
@@ -8,6 +8,7 @@ jdk: openjdk8
 before_install:
 - "./src/main/scripts/ci/before-install.sh"
 - "./src/main/scripts/cd/before-deploy.sh"
+script: "mvn verify -B"
```

Comment:
does it really need?

Reply:
Yes. 

By default, Travis `script` phase contains `mvn test -B` command which runs `maven-surefire-plugin:test`.

So, to run also integration tests using `maven-failsafe-plugin` need to run `verify` phase.